### ISSUE TITLE
Got an error with the globe in a Next.JS project

### ISSUE THREAD

**Issue by MatteoGauthier** [vasturiano/react-globe.gl#46]
**Describe the bug**
I got this error
```
globe.gl.module.js:595 Uncaught TypeError: Cannot read property '__globeObjType' of null
    at globe.gl.module.js:595
    at Function.tick (three-render-objects.module.js:258)
    at Function.comp.<computed> [as tick] (kapsule.module.js:159)
    at Function._animationCycle (globe.gl.module.js:373)
    at comp.<computed> (kapsule.module.js:159)
```

**To Reproduce**

https://codesandbox.io/s/quizzical-moore-59x4h

**Expected behavior**
It should work

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 90

**Comment by vasturiano** [comment]
@MatteoGauthier thanks for creating that example. I was unable to reproduce your issue in that link though. Are you still experiencing the problem?

Also, I noticed you're using `globe.gl` instead of the react module `react-globe.gl`. Any particular reason for that?

**Comment by dylancm4** [comment]
I get the exact same problem periodically in my project dfinityexplorer.org, which uses react-globe.gl. I don't have example code to reproduce the problem, but I thought I'd chime in since I arrived here while searching for a solution to the problem.

```
globe.gl.module.js:595 Uncaught TypeError: Cannot read property '__globeObjType' of null
    at globe.gl.module.js:595
    at Function.tick (three-render-objects.module.js:258)
    at Function.comp.<computed> [as tick] (kapsule.module.js:159)
    at Function._animationCycle (globe.gl.module.js:373)
    at comp.<computed> (kapsule.module.js:159)
```

This causes the globe to lock up. Line 595 looks like this:

`var objType = globeObj.__globeObjType;`

**Comment by vasturiano** [comment]
Thanks to @olaszakos 's PR this issue is now fixed in the latest version (`2.18.1`).

**Comment by MatteoGauthier** [comment]
Thanks a lot !
